src/resources/postcss/views/full/_event-status.pcss

```diff
@@ -0,0 +1,107 @@
+/* -----------------------------------------------------------------------------
+* Event Status.
+* ----------------------------------------------------------------------------- */
+
+.tribe-events-status-text,
+.tribe-common .tribe-events-status-text {
```

Comment:
Any reason why we're adding `.tribe-events-status-text` as a selector as well as `.tribe-common .tribe-events-status-text`? Same question for the other selectors below.

Reply:
No clue it comes from the extension and I was working on cleaning it up.

Reply:
I will see what I can do to only add the selector once.

Reply:
Thanks for bringing this up. I was able to remove all the duplicate selectors with tribe-common.